astropy/wcs/docstrings.py

```diff
@@ -130,7 +139,9 @@ def RA_DEC_ORDER(indent=0):
 
 InvalidTransformError
     Ill-conditioned coordinate transformation parameters.
-""".format(ORIGIN())
+""".format(
+    ORIGIN()
+)
```

Comment:
Does this really need to be on 3 lines? I do not see anything wrong with the old formatting and the new one seems to me unnecessarily ugly and more difficult to read.